integration_test/src/test_desc.rs

```diff
@@ -104,8 +104,10 @@ pub fn test_desc_satisfy(cl: &Client, testdata: &TestData, desc: &str) -> Witnes
     let addr = cl
         .get_new_address(None, Some(json::AddressType::Bech32))
         .unwrap();
+    // Had to decrease 'value', so that fees can be increased 
```

Comment:
Whiespace here. Will merge this now, but please take of this in future. Or address in a later PR.